.github/ISSUE_TEMPLATE/bug_report.yaml

```diff
@@ -39,6 +39,13 @@ body:
     validations:
       required: true
 
+  - type: input
+    attributes:
+      label: Guava Version
```

Comment:
I put this at this position to be close to the "Packages" selection. But maybe it would be better to move it further up (maybe even as first field)?

Reply:
Maybe we can also add a (required?) checkbox to say something like

> - [ ] This is currently [the latest version of Guava available](https://github.com/google/guava/releases/latest).

Reply:
Should this be part of the "Checklist" below then? Or / and should the description of the version field also mention this?

Reply:
The checkbox will show up in the issue once it's filed, so it might be a little weird to put it at the top. I think we can add some instructions at the top asking them to first make sure the bug is still present in the latest version. The checkbox can be at the bottom.